lib/Slider.js

```diff
@@ -45,7 +45,7 @@ var Slider = function (_React$Component) {
   function Slider(props) {
     _classCallCheck(this, Slider);
 
-    var _this = _possibleConstructorReturn(this, Object.getPrototypeOf(Slider).call(this, props));
+    var _this = _possibleConstructorReturn(this, (Slider.__proto__ || Object.getPrototypeOf(Slider)).call(this, props));
```

Comment:
What is the reason for this change? I don't think `__proto__` has good IE support… could be wrong though.